sass/components/_redButton.scss

```diff
@@ -0,0 +1,15 @@
+.redButton {
```

Comment:
Co w sytuacji jak bedziemy chcieli zmienić kolor przycisku na kolor?
Bedziemy musieli wszędzie w wielu plikach w wielu miejscach zmieniać nazwę na "boysenberry", a co jesli klient ponownie zmieni zdanie?
Dlatego w takiej sytuacji lepiej nazwać go "special" lub "awarded"

Reply:
Product Owner w ramach zadań przewidział utworzenie komponentu Przycisk (#13) oraz Przycisk czerwony (#13). Zgodnie z jego wymaganiami nazwa komponentu odpowiada nazwie zadania. Na przyszłość możemy zwrócić uwagę na to, aby taski i tworzone w ich ramach komponenty odpowiadały bardziej ich funkcji a nie wyglądu.

Reply:
@DomiZet Niestety osoba nietechniczna nie zaplanuje dokładnie podziału takiego, żeby kod był przemyślany / rozszerzalny. Coraz bardziej nawet się dąży do tego, że to zespół programistów definiuje taski, do czego dojdziemy niebawem :)

Reply:
Hej, w wymaganiach nie jest podane, że nazwa komponentu ma nazywać się "PrzyciskCzerwony.js". ;)
Tak samo jak "Logo Star Wars" nie nazywa się "LogoStarWars.js". itp.
Technicznie (jako Tech Leader) uważam, że nie trzyma się to konwencji i należy to dostosować.

Reply:
@nowakprojects  - dodatkowo na przyszłość powinniśmy utworzyć według BEM
"button", a on powinien mieć możliwość klasy "button--special".
Choć rozumiem, że tutaj został stworzony taki task, abyśmy mogli się wykazać w kilka osób :)

Reply:
@PiotrWR Masz rację :) I tak - taski były tak, żeby jak najwięcej osób mogło zrobić - ewentualnie 2 różne komponenty na Button - potem się ujednolici.

Reply:
@nowakprojects To miejsce.